code/obj/artifacts/artifact_objects/cloner.dm

```diff
@@ -0,0 +1,105 @@
+
+/obj/artifact/cloner
+	name = "artifact cloner"
+	associated_datum = /datum/artifact/cloner
+
+/datum/artifact/cloner
+	associated_object = /obj/artifact/cloner
+	rarity_class = 4
+	min_triggers = 2
+	max_triggers = 2
+	validtypes = list("ancient","martian","wizard","eldritch","precursor")
+	validtriggers = list(/datum/artifact_trigger/carbon_touch,/datum/artifact_trigger/silicon_touch)
+	react_xray = list(15,90,90,11,"HOLLOW")
+	validtypes = list("wizard","eldritch")
+	touch_descriptors = list("You seem to have a little difficulty taking your hand off its surface.")
+	var/mob/living/carbon/human/clone = null
+	var/imprison_time = 0
+	var/evil_delay = 0
+	var/swapSouls = FALSE
+
+	New()
+		..()
+		imprison_time = rand(50,1200)
+		evil_delay = rand(0,imprison_time)
+		swapSouls = pick(TRUE, FALSE)
+
+	effect_touch(var/obj/O,var/mob/living/user)
+		if (..())
+			return
+		if (!user)
+			return
+		if (clone)
+			return
+		if (ishuman(user))
+			var/mob/living/carbon/human/H = user
+			// fluff
+			if(swapSouls)
+				boutput(user, "<span class='alert'>You feel your soul being sucked out of your body by [O]!</span>")
+			var/filter = filter(type="outline", size=0.5, color=rgb(255,0,0,128), flags=OUTLINE_SHARP)
+			H.filters += filter
+			SPAWN_DBG(0.7 SECONDS)
+				H.filters -= filter
+
+			// a bunch of stolen cloner code
```

Comment:
I like Yass's suggestion that with a small chance it could make a complete copy of the mob by doing
```
clone = semi_deep_copy(H)
```
instead. (This will copy all their items and everything.)

Reply:
I took that as more of a joke suggestion.^^
Is it really ok and won't cause any issues?

Reply:
It might cause issues but I think you should go for it anyway!